frontend/components/Stepper/Stepper.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext, useEffect } from "react";
+import { FormContext } from "../Overlay/CreateProject.js";
+import classes from "../../styles/Stepper.module.css";
+
+function Stepper() {
+  const { activeStepIndex } = useContext(FormContext);
+  useEffect(() => {
+    const stepperItems = document.querySelectorAll(`.${classes.step}`);
+    stepperItems.forEach((step, i) => {
+      if (i <= activeStepIndex) {
+        step.classList.add(`${classes.stepperItemFormat}`);
+      } else {
+        step.classList.remove(`${classes.stepperItemFormat}`);
+      }
+    });
+  }, [activeStepIndex]);
+  return (
+    // <div className="w-2/3 flex flex-row items-center justify-center px-32 py-16">
+    <div className={classes.stepper}>
+      <div className={classes.step}>1</div>
+      <div className={classes.line}></div>
+      <div className={classes.step}>2</div>
+      <div className={classes.line}></div>
+      <div className={classes.step}>3</div>
+      <div className={classes.line}></div>
+      <div className={classes.step}>4</div>
+      <div className={classes.line}></div>
+      <div className={classes.step}>5</div>
+    </div>
+  );
```

Comment:
Instead of the useEffect above that sets the class of the active step with `querSelectorAll` and `classList` functions, you can set the active class directly with React instead....

```suggestion
  return (
    // <div className="w-2/3 flex flex-row items-center justify-center px-32 py-16">
    <div className={classes.stepper}>
      <div className={`${classes.step} ${0 <= activeStepIndex ? classes.stepperItemFormat : ''}`}>1</div>
      <div className={classes.line}></div>
      <div className={`${classes.step} ${1 <= activeStepIndex ? classes.stepperItemFormat : ''}`}>2</div>
      <div className={classes.line}></div>
      <div className={`${classes.step} ${2 <= activeStepIndex ? classes.stepperItemFormat : ''}`}>3</div>
      <div className={classes.line}></div>
      <div className={`${classes.step} ${3 <= activeStepIndex ? classes.stepperItemFormat : ''}`}>4</div>
      <div className={classes.line}></div>
      <div className={`${classes.step} ${4 <= activeStepIndex ? classes.stepperItemFormat : ''}`}>5</div>
    </div>
  );
```
Or even better you could create a `StepCount` component that takes a `isDone` prop and you would do something like...
```
  return (
    // <div className="w-2/3 flex flex-row items-center justify-center px-32 py-16">
    <div className={classes.stepper}>
      <StepCount isDone={0 <= activeStepIndex}>1</Step>
      <div className={classes.line}></div>
      <StepCount isDone={1 <= activeStepIndex}>2</Step>
      <div className={classes.line}></div>
      <StepCount isDone={2 <= activeStepIndex}>3</Step>
      <div className={classes.line}></div>
      <StepCount isDone={3 <= activeStepIndex}>4</Step>
      <div className={classes.line}></div>
      <StepCount isDone={4 <= activeStepIndex}>5</Step>
    </div>
  );
```